vocode/streaming/synthesizer/azure_synthesizer.py

```diff
@@ -364,16 +364,19 @@ def ready_synthesizer(self):
     def get_message_up_to(
         self,
         message: str,
-        ssml: str,
+        ssml_parts: list[str],
         seconds: float,
         word_boundary_event_pool: WordBoundaryEventPool,
     ) -> str:
+        if len(ssml_parts) > 1:
```

Comment:
you're only appending non-dtmf parts to `ssml_parts`, so I think this will generally work. Am I missing something?

it WILL break if a message starts or ends with a DTMF tone. But, like, half the code in this file would also break, so I don't think we need to worry about it in this ticket.

Reply:
So we tested ssml + dtmf + ssml with that agent I made. We made the assumption that repeated ssml fragments is what breaks the word event boundary. Its a reasonable assumption. 

The point I'm bringing up is that we could have a message that ends with a dtmf tone and we'd still try and use the word event boundary. If our assumption was correct its fine.